sales/js/app.js

```diff
@@ -1,15 +1,13 @@
 'use strict';
 
-
 var locationsInfo = [
 ['1st and Pike' , 23 , 65 ,6.3],
-['SeaTac Airport', 3 ,24 , 1.2],
-['Seattle Center', 11, 38 , 3.7],
-['Capitol Hill', 20,38 , 2.3],
-['Alki', 2 ,16 , 4.6],
+// ['SeaTac Airport', 3 ,24 , 1.2],
```

Comment:
Make sure to comment these back in before turning in HW! Also try not to merge commented out code into master!